src/planner/expression_binder/column_alias_binder.cpp

```diff
@@ -42,7 +42,11 @@ bool ColumnAliasBinder::BindAlias(ExpressionBinder &enclosing_binder, unique_ptr
 }
 
 bool ColumnAliasBinder::DoesColumnAliasExist(const ColumnRefExpression &colref) {
-	return bind_state.alias_map.find(colref.column_names.back()) != bind_state.alias_map.end();
+	if (!colref.IsQualified()) {
```

Comment:
We now support `alias.[alias_name]` so this is not correct. This should be `IsPotentialAlias(colref)` and stay as `colref.column_names.back()` - we could add this as an early out and keep the earlier code:

```
if (!IsPotentialAlias(colref)) {
    return false;
}
return ...
```